awx/api/views/__init__.py

```diff
@@ -3391,7 +3391,7 @@ def post(self, request, *args, **kwargs):
         obj = self.get_object()
         if obj.can_cancel:
             obj.cancel()
-            schedule_task_manager()
+            ScheduleWorkflowManager().schedule()
```

Comment:
did you import the right thing in this file? I see `ScheduleTaskManager` imported here